notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T232-analysis_actuators.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>Can we define the dates and any other user input data at the beginning of the notebook? I believe it will be useful if we can set it up in a semi-automatic way (hopefully people can execute the cells without thinking much). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done